.eslintrc

```diff
@@ -1,76 +1,80 @@
 {
-  "parser": "babel-eslint", // now for the support of allowImportExportEverywhere
+  "parser": "babel-eslint",
+  // "extends": [
+  //   "plugin:jsx-a11y/recommended"
+  // ],
   "env": {
     "browser": true,
-    "node": true,
     "es6": true,
-    "mocha": true
+    "jest/globals": true,
+    "mocha": true,
+    "node": true
   },
   "globals": {
-    "browser": true,
     "Assets": true, // Meteor global
     "Package": true, // Meteor global
     "FS": true, // gridFS,
-    "Alerts": true,  // this needs to get cleaned up, but ignoring for now
-    "jest": true,
-    "expect": true,
-    "test": true
+    "Alerts": true  // this needs to get cleaned up, but ignoring for now
   },
-  "parserOptions": {
-     "ecmaVersion": 6,
-     "sourceType": "module",
-      "ecmaFeatures": {
-        "arrowFunctions": true,
-        "blockBindings": true,
-        "classes": true,
-        "defaultParams": true,
-        "destructuring": true,
-        "forOf": true,
-        "generators": false,
-        "modules": true,
-        "objectLiteralComputedProperties": true,
-        "objectLiteralDuplicateProperties": false,
-        "objectLiteralShorthandMethods": true,
-        "objectLiteralShorthandProperties": true,
-        "spread": true,
-        "superInFunctions": true,
-        "templateStrings": true,
-        "jsx": true
-      },
-     "allowImportExportEverywhere": true
+  "settings": {
+    "import/resolver": "meteor"
   },
-  "plugins": ["react"],
-  // NOTE: We're now using eslint-4
-  "rules": {
+  "parserOptions": {
+    "ecmaVersion": 2017,
```

Comment:
Diff looks weird here, but the only actual change to `parserOptions` is changing ecmaVersion to 2017